mlflow/store/file_store.py

```diff
@@ -7,7 +7,7 @@
 import six
```

Comment:
It is not experimental - the meta.yaml that's written just calls `dict` on the Experiment object, which, naturally, does not have any tags in it at the time of creation. At read time, we just poll for the new tags that were written and we override that parameter. I went ahead and just added code to manually delete this from the written dictionary, but I don't think this should cause any bugs otherwise.